scripts/joinmarket-qt.py

```diff
@@ -347,12 +358,7 @@ def checkAddress(self, addr):
             self.bip21_uri = None
 
         self.addressInput.setText(addr)
-        valid, errmsg = validate_address(str(addr))
-        if not valid:
-            JMQtMessageBox(self,
-                       "Bitcoin address not valid.\n" + errmsg,
-                       mbtype='warn',
-                       title="Error")
+        checkAddress(addr)
```

Comment:
should be `self.checkAddress(addr)` (this kind of thing is immediately found by running on regtest :) ).

(Btw this refactoring into two functions looks fine)

Reply:
D'oh!